modules/nw-install-contour-operator.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * networking/about-contour-operator.adoc
+
+[id="nw-install-contour-operator_{context}"]
+= Installing Contour Operator
+
+Install Contour Operator on {product-title} to use link:https://gateway-api.sigs.k8s.io/[Gateway API].
+
+.Prerequisites
+
+* You installed an {product-title} cluster and the `oc` command line.
+//* Log in as a user with cluster-admin permission.
+
+.Procedure
+
+. Install Contour Operator:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/projectcontour/contour-operator/v1.16.0/examples/operator/operator.yaml
+----
++
+[NOTE]
+====
+It can take a few minutes for the Contour Operator to become available.
+====
+
+. Verify the availability of the Operator:
++
+[source,terminal]
+----
+$ oc get deployment/contour-operator -n contour-operator
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME               READY   UP-TO-DATE   AVAILABLE   AGE
+contour-operator   1/1     1            1           12m
+----
+
+. Add `contour` and `contour-certgen` service accounts to the `nonroot` scc:
++
+[source,terminal]
+----
+$ oc adm policy add-scc-to-user nonroot system:serviceaccount:projectcontour:contour
+----
++
+[NOTE]
+====
+The example uses `projectcontour` by default as the namespace of the `contour/contour-certgen` service accounts. Replace `projectcontour` with the namespace used for Contour if you deviate from the example.
+====
```

Comment:
Maybe move this note before the first command as it applies to both commands.

Reply:
Done!